docs/content/usage/repo-mirror.en-us.md

```diff
@@ -58,7 +58,7 @@ The repository now gets mirrored periodically to the remote repository. You can
 
 To set up a mirror from Gitea to GitHub, you need to follow these steps:
 
-1. Create a [GitHub personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with the *public_repo* box checked.
+1. Create a [GitHub personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with the *public_repo* box checked. Also check the **workflowUpdate** checkbox in case your repo using act for continuous integration.
```

Comment:
Where is the `workflowUpdate` ?

## Tokens (classic)

![image](https://github.com/go-gitea/gitea/assets/2114189/80e2ed77-ea0a-4651-9e29-97edacde55dc)

## New fine-grained personal access token

![image](https://github.com/go-gitea/gitea/assets/2114189/a695af62-179e-42c7-9896-d3d8f993305f)

Reply:
Yep